widgets/text/src/widget.tsx

```diff
@@ -1,73 +1,90 @@
-// @ts-nocheck File not migrated fully to TS
 export {};
 
+const t = Stage.Utils.getT('widgets.text');
+
+type Configuration = {
+    header: string;
+    content: string;
+    headerTextFont: string;
+    headerTextColor: string;
+    headerTextSize: number;
+    contentTextFont: string;
+    contentTextColor: string;
+    contentTextSize: number;
+};
+
 const textFonts = [
     { name: 'Sans Serif', value: 'sans-serif' }, // used as default
     { name: 'Serif', value: 'serif' },
     { name: 'Monospace', value: 'monospace' }
 ];
 
-Stage.defineWidget({
+Stage.defineWidget<unknown, unknown, Configuration>({
```

Comment:
I guess in case of this widget we are sure that data is not fetched and we don't provide any params, so probably more accurate would be to define it like:
```suggestion
Stage.defineWidget<never, never, Configuration>({
```

Reply:
I changed to `never`: [b70f39c](https://github.com/cloudify-cosmo/cloudify-stage/pull/2216/commits/b70f39cde6c2191c9bb78829ff79615132113abc)